src/averageAges.js

```diff
@@ -15,6 +15,16 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
+  const men
+    = arguments.length > 1
+      ? people.filter(person => Math.ceil(person.died / 100) === century
+        && person.sex === 'm')
+      : people.filter(person => person.sex === 'm');
```

Comment:
No need to duplicate `filter`, just choose needed filter condition inside `filter` callback